internal/types/fjson/fjson_test.go

```diff
@@ -154,78 +123,3 @@ func testJSON(t *testing.T, testCases []testCase, newFunc func() fjsontype) {
 		})
 	}
 }
-
-func fuzzJSON(f *testing.F, testCases []testCase, newFunc func() fjsontype) {
-	for _, tc := range testCases {
-		f.Add(tc.j)
-		if tc.canonJ != "" {
-			f.Add(tc.canonJ)
-		}
-	}
-
-	f.Fuzz(func(t *testing.T, j string) {
-		t.Parallel()
-
-		// raw "null" should never reach UnmarshalJSON due to the way encoding/json works
-		if j == "null" {
-			t.Skip()
-		}
-
-		// j may not be a canonical form.
-		// We can't compare it with MarshalJSON() result directly.
```

Comment:
As per this, removing this fuzz test.

Reply:
Let's add a call to `testutil.Dump` that uses `fjson` to `bson`'s package fuzzing. That way we would ensure that we could log all BSON values

Reply:
This might be a bit overkill for this case actually. It seems `bson` needs to be `types.Type` for `testutil.Dump` to be happy and we'd need to handle the conversion to each `types.Type`. Perhaps, I could have missed some utility function that might do that. Anyway I'm happy to leave this as it is for now, WDYT? :thinking:

Reply:
Sounds good